### PR TITLE
Update draft-toutain-schc-access-control-00.md

### PR DESCRIPTION
**Pull request by MarinoMtz**
Adding content on the architecture part and adding some changes on the document structure

### CODE REVIEW

File: draft-toutain-schc-access-control-00.md [hunk 1]
```diff
@@ -47,45 +47,61 @@ informative:
...
-   (--------)     |processing|        +-------+    +---------------+
-                  +----------+
+     ......................................................
+     .   ....................................             .
+     v   ^     create                       v             ^   
+   (--------)  read    +=======+     +----------+    +-------+    +-------+------+
+   ( Set of )<-------->|Rule   |<--->|Management|<===|Access |<===|Other end     |<=== 
+   ( Rules  )  update  |Manager|     |request   |    |Control|    |authentication| Management 
+   (--------)  delete  +=======+     |processing|    +-------+    +--------------+ Request
+                                     +----------+                                  NETCONF, RESTCONF or CORECONF
```
[the remark this answers is not in the document]
Comment (MarinoMtz) [reply draft-toutain-schc-access-control-00.md:76]: which is the top rule?, the one between access control and set of rules? I don't get your point, the only change I made is to put the Rule Manager in the middle

[the remark this answers is not in the document]
Comment (MarinoMtz) [reply draft-toutain-schc-access-control-00.md:76]: Ah I see your point now, I just forget to move the starting point of the arrow, I'll make the change, and add this point inside the explanation.

[the remark this answers is not in the document]
Comment (MarinoMtz) [reply draft-toutain-schc-access-control-00.md:76]: But still, retrieving the corresponding rule is not done at the authentication phase, imho it shall be done at the Access Control

**Comment by minaburo** [comment]
Hello Guys,

Please tell me where is the last version of this document? I cannot find in
the LPWAN GitHub any version nor a directory with this document. Did you
create a SCHC GitHub?

Thanks
Ana

**Pull request merged**